### ISSUE TITLE
Redundant Steps on Step 5 when making a new repo, syncing it and such

### ISSUE THREAD

**Issue by Squints84** [open-learning-exchange/open-learning-exchange.github.io#3328]
### Please verify the following:

- [X] This issue is related to the [Mobile Interns First Steps](https://open-learning-exchange.github.io/#!pages/mi/mi-10-steps.md) document.
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Ami) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!./pages/mi/mi-github-and-repositories.md

### Problem

Here on step 5, I feel that the steps to sync a repository take away from the later parts of the page where it goes into more in-depth. It may be good as a quick intro to the later sections that go into more detail, but I think it really just ends up undermining the detailed explanations.

### Screenshots (when necessary)

![image](https://github.com/user-attachments/assets/b35c9b8f-20e5-4cab-a507-a41d919ee948)

### Proposed solution(s)

I think moving this set of steps to the 'Summary of Steps' section would be a good idea in order to highlight the detailed sections mores.

**Comment by xyb994** [comment]
I actually noticed this issue last week. It was introduced by a prospective mobile intern meant to simplifying something but ended up creating a summary and we did not catch that because we did not read the whole page when reviewing the changes.

I believe the content in 2.1 are pretty much already in the summary of steps. Go ahead and create a pull request to adopt what's in 2.1 into the summary then remove it altogether. I will work on cleaning up section 3 and 4 a little bit and ask you to take a look later :)
